app/javascript/elm/src/FixedSessionFilters.elm

```diff
@@ -101,7 +114,8 @@ updateLabels msg model toSubCmd mapper updateModel =
 view : Model -> Html Msg
 view model =
     div []
-        [ Html.map ProfileLabels <| LabelsInput.view model.profiles "Profile Names" "profiles-search"
+        [ LocationFilter.view model.location UpdateLocationInput SubmitLocation
```

Comment:
Have you considered keeping that in this file? In my experience with Elm it's better to move to a new file only if there's a strong argument. 

I know this is gonna be used by mobile too but we have duplication in any case. We will hopefully solve that when merging the two